.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+# This script is executed whenever the docker container is (re)started.
+
+#===============================================================================
+# debugging
+set -x
+
+#===============================================================================
+# environment
+reentry scan
+
+export SHELL=/bin/bash
+
+#===============================================================================
+# setup AiiDA
+aiida_backend=django
+
+if [ ! -d /home/$SYSTEM_USER/.aiida ]; then
+    verdi setup                                \
+        --profile default                      \
+        --non-interactive                      \
+        --email some.body@xyz.com              \
+        --first-name Some                      \
+        --last-name Body                       \
+        --institution XYZ                      \
+        --db-backend $aiida_backend            \
+        --db-username aiida                    \
+        --db-password aiida_db_passwd          \
+        --db-name aiidadb                      \
```

Comment:
I take it these values correspond to what is hard-coded in `aiida-prerequisites` where a postgres DB is already installed. Seems a bit delicate, no? Would it be better to have database creation in same docker layer?

Reply:
I agree - this would also also let us switch to `verdi quicksetup` here, so we'd be testing that it actually works.

Reply:
Made this configuration optional.